Core/src/org/sleuthkit/autopsy/rejview/RejTreeKeyView.java

```diff
@@ -95,7 +102,12 @@ public RejTreeKeyView(RejTreeKeyNode node) {
             numValues = -1;
         }
 
-        JLabel metadataLabel = new JLabel(String.format(metadataTemplate, keyName, numSubkeys, numValues), JLabel.LEFT);
+        Date date = new java.util.Date(node.getKey().getTimestamp().getTimeInMillis()); 
```

Comment:
Is there any chance getTimestamp() could return null? It doesn't look like it to me but I'm not sure.